### PR TITLE
Adding tool to extract clusters from the vcf file

### PR DESCRIPTION
**Pull request by ffinfo**
### Description

_Give your PR a **concise** yet **descriptive** title_

----

### Checklist (never delete this)

- [ ] Each method/class/object/trait should have scaladocs
- [ ] Added methods are unit tested
- [ ] Test coverage may not drop
- [ ] Documentation should be updated if required

### CODE REVIEW

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package nl.biopet.tools.tenxkit
+
+case class GenotypeCall(genotype: Array[Option[Int]],
+                        ad: Array[Int],
+                        aq: Array[Int],
+                        pl: Array[Array[Int]])
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala]: ad, aq, pl ? 
Meaning? Informative names would be nice.

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+object GenotypeCall {
+
+  def probabilityToPhred(p: Double): Int = (-10 * math.log(p)).toInt
+
+  private val t = 2.0
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala:40]: unless `t` means time in seconds, can you give an informative name?

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                      ploidy: Int = 2): Double = {
+    val v = -math.pow(
+      (fraction * (ploidy.toDouble / 2)) * t - ((t / 2.0) * expectedAlleles),
+      2.0) + 1.0
+    if (v > 1.0 || v < 0.0) 0.0 else v
+  }
+
+  /** Generates a [[GenotypeCall]] from a given allele depths */
+  def fromAd(ad: Array[Int], ploidy: Int = 2): GenotypeCall = {
+    val totalDepth = ad.sum
```
Comment (rhpvorderman) [inline src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala]: ad?

**Review by rhpvorderman — CHANGES_REQUESTED** [review]

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala (hunk 2 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala:40]: no idea here, it's just a fixed number in the formula, it's private for a reason ;)

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala (hunk 3 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala]: Added scaladocs

File: src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala (hunk 1 above)
Comment (ffinfo) [reply src/main/scala/nl/biopet/tools/tenxkit/GenotypeCall.scala]: Added scaladocs

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**